FrEIA/modules/base.py

```diff
@@ -0,0 +1,38 @@
+from typing import Tuple, Iterable, Union
+
+import torch.nn as nn
+from torch import Tensor, is_tensor
+
+
+class InvertibleOperator(nn.Module):
+    def __init__(self, dims_in: Tuple[int], dims_c: Tuple[int] = None):
```

Comment:
doc string is missing

Reply:
also, dims_in is is a `Tuple[Tuple[int]]`, because of multiple inputs! (another problem IMO with the 'sometimes tuple sometimes not'-approach.
Because now you need extra code 
```python
if isinstance(dims_in[0], tuple):
    tuple_input_dimension = True
# etc
```
Analogous to `.forward()`

Reply:
see comment above (still working out this 'code review' feature...)